src/modules/hunting/dashboard.py

```diff
@@ -23,6 +23,7 @@ def __init__(self, event):
         self.event = event
 
     def get_nodes(self):
+        logging.debug("Active hunter is attempting to get nodes types of the cluster")
```

Comment:
Not quite English - how about "Active hunter is attempting to get node information"?